testsuite/tests/libratest/smoke_test.rs

```diff
@@ -360,6 +361,20 @@ fn test_startup_sync_state() {
     // during a node startup
     fs::remove_dir_all(state_db_path).unwrap();
     assert!(env.validator_swarm.add_node(peer_to_stop, false).is_ok());
+    retry(Fixed::from_millis(10_000).take(5), || {
```

Comment:
Lets rescale it to retry more often so test pass faster? something like 50 times every second wait?

Reply:
👍